README.rst

```diff
@@ -1,11 +1,24 @@
 PyFFI
 =====
-.. image:: https://travis-ci.org/niftools/pyffi.png?branch=develop
+.. image:: https://img.shields.io/travis/niftools/pyffi/develop.svg
```

Comment:
How were these created, who controls them?

Reply:
https://shields.io is a website that provides high-quality metadata badges for open source projects.